### ISSUE TITLE
A word of warning should be required

### ISSUE THREAD

**Issue by jsanta** [thatisuday/catage#2]
Not precisely an issue but I see a need of a work of warning before installing the package. 
Catage requires Puppetteer, which is a 114MB package, which is quite large if this package is mistakenly included in any production deploy that doesn't really requires it.

Nice work by the way. even though I have mixed feelings about using images instead of copy & pastable code in articles.
Best regards.

**Comment by thatisuday** [comment]
@jsanta Thanks for the feedback. Let me clarify why we used [**puppeteer**](https://github.com/GoogleChrome/puppeteer). The `catage` package is basically a combination of multiple libraries. First, we convert code to ANSI color formatted text and then convert ANSI color formatted text to SVG. To convert SVG to an Image, we need a renderer like a headless web browser.  Chromium is open-source browser and Google provides JavaScript API to interact with it through their **puppeteer** package. We convert SVG to a Data URI and open it in the headless Chromium browser. Then we take a screenshot of the page and close the browser. This screenshot is nothing but the final image saved on the disk.

You can include code in the article but once it gets large and there are too much of them in an article, plugins or embeds starts to mess around with the performance of the web browser. Besides, I have included links to the code below every code snippet. Also, having result of the code execution is hard to embed and most plugins do not support it. I can speak for myself, I didn't like even a single one out there.

Hope this information helped you clarify your doubts.

**Comment by jsanta** [comment]
Thanks for your answer.
I understand  the need for Puppetteer, I've used it myself to generate images and pdf files. What I say is that  even though a couple of GB is _nothing_ in terms of costs, if this library is mistakenly included in any production deploy, it will be an extra of 100MB +- so any word that helps us being aware of  _"Hey your going to install a 112MB dependency with this package"_, it will nice.

Basically this would be just to avoid silent _"Nobody told me this package weights more than a 100MB"_ comments, considering most developers don't check the dependencies of every package they ``npm install --save ``

**Comment by thatisuday** [comment]
Done. Updated README.md

**Comment by m00nwtchr** [comment]
Couldn't you use a svg-renderer library for this instead? e.g. this one https://www.npmjs.com/package/svg-render
